SharedPackages/BrowserServicesKit/Sources/Subscription/Managers/V1toV2Bridging/SubscriptionAuthV1toV2Bridge.swift

```diff
@@ -54,15 +61,19 @@ public protocol SubscriptionAuthV1toV2Bridge: SubscriptionTokenProvider, Subscri
 }
 
 extension SubscriptionAuthV1toV2Bridge {
-
-    public func isEnabled(feature: Entitlement.ProductName) async throws -> Bool {
-        try await isFeatureAvailableAndEnabled(feature: feature, cachePolicy: .returnCacheDataElseLoad)
+    public func isFeatureIncludedInSubscription(_ feature: Entitlement.ProductName, cachePolicy: APICachePolicy) async throws -> Bool {
+        try await currentSubscriptionFeatures().contains(feature)
     }
 }
 
 extension DefaultSubscriptionManager: SubscriptionAuthV1toV2Bridge {
 
-    public func isFeatureEnabledForUser(feature: Entitlement.ProductName) async throws -> Bool {
+    public func isFeatureEnabled(_ feature: Entitlement.ProductName) async throws -> Bool {
+
+        // Note by Diego: we should probably check if the feature is included in the subscription
+        //   here, but I'm trying to minimize the amount of changes, and up until today I see the
+        //   code is only checking entitlements.  I'll keep that logic for now, but we should
+        //   try to understand if this is important to change.
```

Comment:
As discussed with @federicocappelli , I'm not checking if the feature is included in the subscription because that could introduce troublesome regressions.